cms/apps/media/models.py

```diff
@@ -169,19 +170,27 @@ def formfield(self, **kwargs):
 }
 
 
-class ImageRefField(FileRefField):
+class ImageRefField(models.ForeignKey):
```

Comment:
Is there a reason the inheritance from `FileRefField` was changed? It looks like it duplicates a few lines of code from it as a consequence of removing this ancestor.

Reply:
No, it was just erroring. Inheritance has now been added back with a fix.